### DISCUSSION TITLE
Speed Comparison Sata vs. PCIE

### DISCUSSION THREAD

**Discussion by Simonstroe** [R3NE07/Futro-S740#29] [Q&A] [answered]
Hi together

Has anyone ever compared what actual write and read rates can be achieved with Sata or PCIE x1 on the Futro?
The background is that I want to build a setup with only one internal hard drive and use the fastest available port for it.
Or does it make sense to set up a Raid 0 to increase the speed? The system should serve as a multi-purpose device for HomeAssistant, but also a local Nextcloud and PhotoPrism. For data security I provide an encrypted off-site backup.

**Comment by Hummtaro** [comment, marked as answer]
Of course, since we only have PCIe 2.0 X1, SATA can be faster.
I've tested ~560 MB/s over SATA and ~450 MB/s over PCIe.
In my Proxmox machine i ended up with two PCIe in BTRFS RAID0, which gave me ~900 MB/s.

**Comment by danzbox** [comment]
I tested a PCI-e SSD Samsung PM9B1 with 256GB in the SSD SATA/PCI-E slot. It was only 250MB/s constant write with H2testw and ~400/350 with Crystaldisk benchmark, 109k/80k IOPs. On a laptop with PCI 4.0 the SSD performed 3100/1000MB/s with Crystal disk mark (SLC cached) so it is not a limit of the size of only 256GB.
